### DISCUSSION TITLE
【建议】希望增加全屏模式下时间进度功能

### DISCUSSION THREAD

**Discussion by shanchengzi** [the1812/Bilibili-Evolved#4055] [功能建议 / Ideas]
全屏播放视频时，如果不频繁滑动鼠标，就无法实时看到视频播放进度，所以，希望增加个选项，可以显示视频总时长、当前播放时长。
虽然我找到别的扩展提供了这项功能，但总体功能不如Bilibili-Evolved丰富，且快捷键冲突。

**Comment by Geeyun-JY3** [comment]
重复，请善用搜索：#2618

**Comment by shanchengzi** [comment]
第一次到这里反馈，因为用过好久也没看到类似功能，还以为没人反馈呢。看了下链接里的，已经是2021年别人反馈的了，但到现在也没有增加这个功能，下面评论的人也只是变相叫提问的把全屏改成网页全屏来看。所以，这个功能以后也不会增加了吗？
搜索了下，没找到能实现这个功能的脚本，扩展倒是有一个，和你链接里的不同，这个是显示在左上角，并且中上还能显示标题。和安卓端的MX&nbsp;Player相似。可惜它和别的扩展脚本有冲突，并且快捷键不像Bilibili-Evolved一样可以修改https://microsoftedge.microsoft.com/addons/detail/%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9bilibili%E6%92%AD%E6%94%BE%E5%99%A8%E6%89%A9%E5%B1%95/kifdlkkkncadbhofogcaggklmgpcipek




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "the1812/Bilibili-Evolved"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月11日(星期六) 下午5:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] 【建议】希望增加全屏模式下时间进度功能 (Discussion #4055)





 
重复，请善用搜索：#2618

**Reply by Geeyun-JY3** [reply]
既然是第一次来这里反馈，那请认真阅读 #1297，里面写得很清楚 discussion 被打的标签是什么意思。
去已有的相关 discussion 下评论也有更大的可能将相关信息传达到相关组件的开发者。（也看到你已经这么做了）
你这需求就有一个相关的现存组件，开发者也不是本 repo owner。
